vagrant_opm_user/bootstrap.sh

```diff
@@ -20,9 +20,9 @@ sudo apt-get update -y
 
 # OPM packages
 sudo apt-get install libopm-core1-bin -y
-sudo apt-get install libopm-autodiff1-bin -y
-sudo apt-get install libopm-polymer1-bin -y
+sudo apt-get install libopm-simulators1-bin -y
 sudo apt-get install libopm-upscaling1-bin -y
+sudo apt-get install openmpi-bin
```

Comment:
because there's no direct dependency. the correct way to do it would be to have a meta package which depends on opm-simulators-bin and mpi-default-bin. it's basically related to full-distro rebuilds, they do not want to rebuild everything that requires mpiexec to run if only the mpiexec binary was changed.